src/containers/SubjectPage/components/SubjectLinks.tsx

```diff
@@ -65,7 +62,7 @@ const SubjectLinkSet = ({ set, subjects, title }: SubjectLinkSetProps) => {
       {subjects.map((subject, index) => (
         <>
           <LinkElement key={`${set}-${index}`}>
-            <StyledLink href={subject.path}>{subject.name}</StyledLink>
+            <SafeLink to={subject.path as string}>{subject.name}</SafeLink>
```

Comment:
Hvis det er en sjanse for at denne kan være undefined burde man ta hensyn til det. Hvis den aldri kan være null burde man oppdatere typen i GQL